build-logic-commons/basics/src/main/kotlin/gradlebuild/basics/BuildParams.kt

```diff
@@ -241,10 +239,6 @@ val Project.buildMilestoneNumber: Provider<String>
     get() = gradleProperty(BUILD_MILESTONE_NUMBER)
 
 
-val Project.isRetryBuild: Boolean
```

Comment:
Do we add a build scan tag for retry builds? I think it would be good if we could do that.

Reply:
We do that already: `RetriedBuild` is added in:
https://github.com/gradle/gradle/blob/63602dd5c49306e0d465ea311b2bd0106f1a7676/.teamcity/src/main/kotlin/configurations/GradleBuildConfigurationDefaults.kt#L125